iam.tf

```diff
@@ -0,0 +1,16 @@
+module "job_execution_role" {
+  count                 = var.role_arn == null ? 1 : 0
+  source                = "github.com/schubergphilis/terraform-aws-mcaf-role?ref=v0.4.0"
+  name                  = "GlueExecution-${var.name}"
+  create_policy         = var.role_policy != null ? true : false
+  principal_type        = "Service"
```

Comment:
policy_arns = ["arn:aws:iam::aws:policy/service-role/AWSGlueServiceRole"]

Reply:
fixed it